app/_locales/en/messages.json

```diff
@@ -3666,6 +3666,9 @@
   "prev": {
     "message": "Prev"
   },
+  "previousApprovedSignedTransaction": {
+    "message": "A previous transaction is still being signed or submitted."
```

Comment:
we don't need a period (single line of content)
```suggestion
    "message": "A previous transaction is still being signed or submitted"
```

Reply:
Hey @matthewwalsh0, @sallykhomikh is our new content designer and we were walking through how she might use GitHub to improve the content of the extension. Your PR just happened to be at the top of the list and had content we could review. [Heres more information via our content guidelines](https://www.notion.so/MetaMask-Content-Design-Style-Guide-7d0b7579786540afbe02780ebb203254?pvs=4#7ffcee3343bc4115991c7a7e0d2f1c71)

![Screenshot 2024-01-26 at 5 03 05 PM](https://github.com/MetaMask/metamask-extension/assets/8112138/93591400-aa3c-4af0-b2ea-7e75626385f0)